guides/common/sass/components/_asciidoc.scss

```diff
@@ -0,0 +1,1489 @@
+// QUESTION: should we add max-width:100% and height:auto to svg?
```

Comment:
Do we have to address this question in this PR? Do you want to create an issue for that?

Reply:
This is copy/pasted file from asci-doc styles, not my comment

Reply:
OK, then disregard my questions.